modules/claims_api/app/swagger/claims_api/v1/swagger.json

```diff
@@ -3766,10 +3772,16 @@
                                 ],
                                 "properties": {
                                   "receiving": {
-                                    "description": "Is Veteran getting Retiree pay.",
+                                    "description": "Is Veteran getting Retiree pay?",
                                     "type": "boolean",
                                     "example": true
                                   },
+                                  "willReceiveInFuture": {
```

Comment:
we were just missing this entirely from our schema documentation i guess?

Reply:
Yeah I guess.
In that doc EVSS gave us (linked in original issue), there are a couple attributes that they explicitly check but don't exist according to our schema.  So there's a couple related tickets that will bring those attributes into existence.